src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -333,14 +381,47 @@ func (r *ClusterReconciler) createBootstrapStatefulSet(
 								},
 							},
 						},
+						{
+							Name:	"configmap-dir",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: configMapName,
+									},
+									DefaultMode:	&configMapDefaultMode,
+								},
+							},
+						},
+						{
```

Comment:
what is this for?

Reply:
`config-dir` is where the base `redpanda.yaml` is copied to and then modified by `rpk` for that particular Pod.